### DISCUSSION TITLE
[helm/starboard-operator] add enable kube-hunter

### DISCUSSION THREAD

**Discussion by swarren83** [aquasecurity/starboard#1092] [Development]
Add `kubernetesHunterEnabled` as an configuration parameter to the starboard-operator helm chart.

```
operator:
  kubernetesHunterEnabled: false
```

If set to `true` the following would be added to the starboard configMap data field:

```
  kube-hunter.imageRef: docker.io/aquasec/kube-hunter:0.6.3
  kube-hunter.quick: false
```

This will allow execution of `starboard scan kubehunterreport` to be executed via the starboard CLI on that cluster.

**Comment by danielpacak** [comment]
The Starboard Operator is not really using Starboard CLI to do it's job. One does not require the other even if they're compatible to some extent. We have #262 to think about best way of integrating KubeHunter with Starboard Operator. Helm is just an installation method and will be updated in scope of the same issue.
